server.go

```diff
@@ -44,6 +47,8 @@ type Server struct {
 	HmacSecret string
 	// A function that is called before the request is handled by viewproxy.
 	PreRequest func(w http.ResponseWriter, r *http.Request)
+  tracingConfig tracing.TracingConfig
```

Comment:
Indentation looks a bit off, can we run `go fmt`?

Reply:
Fixed in db650657d00931e3531d8225979a1489d37034ad